app/service/family.py

```diff
@@ -104,11 +111,32 @@ def update(
     if family is None:
         raise ValidationError(f"Could not find family {import_id}")
 
+    # Get the organisation from the user's email
+    user_org_id = app_user.get_organisation(db, user_email)
+
     # Validate metadata
     org_id = organisation.get_id(db, family.organisation)
+    if org_id != user_org_id:
+        raise ValidationError(
+            f"Current user does not belong to the organisation that owns family {import_id}"
+        )
     metadata.validate(db, org_id, family_dto.metadata)
 
-    if family_repo.update(db, import_id, family_dto, geo_id):
+    # Validate that the collections we want to update are from the same organisation as
+    # the current user and are in a valid format.
+    all_cols_to_modify = set(family.collections).union(set(family_dto.collections))
+
+    id.validate_multiple_ids(all_cols_to_modify)
+
+    collections_in_usr_org = [
```

Comment:
should this be perhaps named `collections_not_in_user_org`

Reply:
Good spot! Forgot to change after changing from not all to any!